### PR TITLE
chore: [EUDIW-153] Update presentation screen

### PR DESCRIPTION
**Pull request by LazyAfternoons**
## Short description
This PR updates the presentation screen aligning it with the [FIGMA](https://www.figma.com/design/IAq7xk1P2iBL1X9zdboMJk/EUDIW---Potential?node-id=1028-2499&node-type=frame&t=l9ABsO6SCmTVzeIm-0).

## List of changes proposed in this pull request
- Add `ItwMarkdown` component to render markdown;
- Add `ItwRequiredClaims` component to render the required claims for the presentation;
- Update `ItwIssuanceCredentialAuthScreen` to use the aforementioned components;
- Update locales;
- (Extra) Shows the `Reset` button only in debug;

## How to test
Try to obtain a credential and check that the screen matches the one in the FIGMA, except for the logos.

### CODE REVIEW

File: ts/features/itwallet/screens/presentation/remote/pid/ItwPrRemotePidDataScreen.tsx [hunk 1]
```diff
@@ -76,116 +54,100 @@ const ItwPrRemotePidDataScreen = () => {
...
-  const RpPreviewView = ({ pid }: { pid: StoredCredential }) => (
-    <SafeAreaView edges={["bottom", "left", "right"]} style={IOStyles.flex}>
+  const ContentView = ({ pid }: { pid: StoredCredential }) => {
+    const claims = parseClaims(pid.parsedCredential);
+    const requiredClaims = claims.map(
+      claim =>
+        ({
+          claim,
+          source: pid.credentialType
+        } as RequiredClaim)
```
Comment (datalek) [inline ts/features/itwallet/screens/presentation/remote/pid/ItwPrRemotePidDataScreen.tsx:64]: **suggestion (non-blocking)**: I'm not sure, but I think we can remove the `as RequiredClaim`

**Review by datalek — COMMENTED** [review]
I left some comments, they are mostly non-blocking, so feel free to ignore them.

File: ts/features/itwallet/screens/presentation/remote/pid/ItwPrRemotePidDataScreen.tsx (hunk 1 above)
Comment (LazyAfternoons) [reply ts/features/itwallet/screens/presentation/remote/pid/ItwPrRemotePidDataScreen.tsx]: Fixed in [ceaa369](https://github.com/pagopa/io-eudiw-app/pull/40/commits/ceaa369f2b20386b8ec60017e2d64d15b22f6e18), thanks.

**Review by datalek — APPROVED** [review]
LGTM!

**Pull request merged**